app/mailers/candidate_mailer.rb

```diff
@@ -13,6 +13,17 @@ class CandidateMailer < ApplicationMailer
   )
   include QualificationValueHelper
 
+  def application_choice_submitted(application_choice)
+    @application_choice = application_choice
+    @candidate_magic_link = candidate_magic_link(application_choice.application_form.candidate)
+    @reject_by_default_date = @application_choice.reject_by_default_at.to_fs(:govuk_date)
```

Comment:
I might be wrong here, but I suspect that this line might not work when the transaction block didn't commit the `reject_by_default_at` column and we use then here (well, unless Rails change the attribute from a transaction before commit - which will be interesting to see). 
In any case maybe it is best pass the inactive date to this method? @JR-G faced many problems last year with this transaction/commit problems, wdyt?